css/index.css

```diff
@@ -76,8 +81,10 @@ header {
 }
 
 header nav {
-    margin-top: 40px; 
-    width: 610px;
+    /*margin-top: 40px; */
+    margin-top: 4.5%;
```

Comment:
As you can probably recall from the meeting yesterday, margin top/bottom would almost always be in px since it is more for aesthetics while left/right will be in percentages.